tor_core/initialize.py

```diff
@@ -262,6 +265,28 @@ def get_heartbeat_port(config):
             return port
 
 
+def configure_modchat(config):
+    if config.modchat_api_url:
+        # creating the connection is much slower than for slack -- it takes about
+        # a second. Instead of doing that every time we need to send a message,
+        # we'll do it once here and just pass the RocketChat instance around.
+        try:
+            config.modchat = RocketChat(
+                os.environ.get('MODCHAT_API_USERNAME', None),
+                os.environ.get('MODCHAT_API_PASSWORD', None),
```

Comment:
Do we want `None` as a default? It'll fail anyway if the credentials are absent so I'd say `os.environ[...]` would be better. Doesn't really matter though.

Reply:
Yes, it will fail anyway, but at least having it set to none and then attempt to authenticate will result in a graceful failure with an error message rather than flat-out exploding. Seems like a better way to do it for local development.